runtime/lua/vim/loader.lua

```diff
@@ -41,6 +43,16 @@ local Loader = {
   },
 }
 
+--- @param path string
+--- @return CacheHash
+--- @private
+function Loader.get_hash(path)
+  if not Loader._hashes[path] then
+    Loader._hashes[path] = uv.fs_stat(path) --[[@as CacheHash]]
```

Comment:
Just a remark here, that we should never cache hashes for non-existing files. So the code as is is good, since it will be `nil` and getting the hash again will always redo an `fs_stat`. We need to keep doing it this way, or otherwise lazy will not be able to bootstrap properly in some cases.

TLDR: all good :)

Reply:
I'll add a comment.